### DISCUSSION TITLE
Feature request - List your requested features here

### DISCUSSION THREAD

**Discussion by frankNiessen** [ORTools4MTEX/ORTools#39] [Feature Request]
Hi all, 

sometimes I get questions regarding certain features that should be integrated into ORTools. If you have a suggestion for a feature, or if you want to contribute with a feature, you can communicate this here. 

Best wishes
Frank

**Comment by wanghaijian217** [comment]
Hi Frank! I have been very interested in ORTools recently, and plotMap_variantPairs. m in the ORTools (src) you developed draws the ratio and fraction of boundary length of martensitic variant pairs, but only V1-V2; V1-V3 (V5); V1-V6; V1-V4; Can the remaining variant pairs be supplemented, and can the calculation method of variant pairs be further explained?The variant pair diagram is Figure 4 in this reference：https://doi.org/10.1007/s40195-020-01186-x; You have answered the relevant questions before, but I want to point out that this is variant paved with V1.
![image](https://user-images.githubusercontent.com/131143952/233360980-5494753f-067e-4468-a5e8-0e474c0da059.png)

**Reply by AzdiarGazder** [reply]
Hi @wanghaijian217,

A new function [**computeGrainPairs.m**](https://github.com/ORTools4MTEX/ORTools/blob/develop/src/computeGrainPairs.m) has been added to the ORTools Developer to help you calculate grain pairs based on variant, packet, Bain group id *OR any other type of user-specified id*. The advantage of the last option is that you can now return grain pair information based on equivalent ids or any other type of id that you define.

Please see [**Example 10**](https://github.com/ORTools4MTEX/ORTools/blob/develop/ORTools_example10.m) for a detailed demonstration on how to use this new function.

Hope this helps with your analysis.

Warm regards,
Azdi

**Comment by huseyinhzl** [comment]
Hi Frank,

Really so thankful for your and your colleagues' efforts here.  I am wondering if it is possible to implement a code that can measure the sizes of martensitic alpha in titanium alloys like in martensitic stainless steel.

**Reply by frankNiessen** [reply]
Did you try whether the martensitic steel code works on Ti? I cannot see why it should not, maybe it would need some minor modifications.

**Reply by huseyinhzl** [reply]
I did it and working though there are still some differences with the one I measured by AztecCrystal. I think it is still better than using AztecCrystal.  Do you think it is worth trying to measure alpha lath dimensions in the case of non-martensitic Titanium alloys with ORTools?

**Reply by frankNiessen** [reply]
Hi Huseyin, 

maybe you can share a dataset with me and I can see what I can do. Based on that we can take a discussion whether it is worthwhile. 

Best wishes
Frank

**Comment by NovakEvana** [comment]
The same request as I asked for [last time](https://github.com/ORTools4MTEX/ORTools/discussions/25). If you've already been working on that, nevermind. You guys doing great work as always!

Thank you.

**Reply by AzdiarGazder** [reply]
Hi @NovakEvana,

I'm working on the best way to introduce Bain group analysis to Mtex / ORTools. There is working code but we still have to hash a few things out together before releasing it to everyone. As @frankNiessen mentioned, we do development work in our free time and/or if it coincides with our research demands, so please be patient with us. It'll definitely be there soon! :)

**Reply by NovakEvana** [reply]
So great to hear that. Thanks Azdiar and Frank for working on that!

**Comment by mugikumar** [comment]
Hi frank, 
      Please develop the code for five-parameter grain boundary distribution for intervariant boundary analysis by the stereological method.

**Reply by frankNiessen** [reply]
Are you looking for [such](https://github.com/ORTools4MTEX/ORTools/blob/develop/doc/Nyyss%C3%B6nen_Gazder_Hielscher_Niessen_2023.pdf) an analysis?

**Reply by mugikumar** [reply]
Hi Frank,
Thanks for your reply, Yes, the same process, but I need a grain boundary plane distribution for PAG, Packet, Block and Lath by the stereological method. Please look into this paper https://doi.org/10.1016/j.actamat.2013.10.010
